plasmapy/formulary/collisions.py

```diff
@@ -111,130 +109,261 @@ def Coulomb_logarithm(
         a macroscopic description is valid. This is used to recover the
         average ion density (given the average ionization and electron
         density) for calculating the ion sphere radius for non-classical
-        impact parameters.
+        impact parameters. `z_mean` is a required parameter if `method` is
+        `"lsfullinterp"`, `"hlsmaxinterp"`, or `"hlsfullinterp"`.
 
     V : ~astropy.units.Quantity, optional
         The relative velocity between particles.  If not provided,
         thermal velocity is assumed: :math:`\mu V^2 \sim 2 k_B T`
         where `mu` is the reduced mass.
 
-    method: str, optional
-        Selects which theory to use when calculating the Coulomb
-        logarithm. Defaults to classical method.
+    method : str, optional
+        The method by which to compute the Coulomb logarithm.
+        The default method is the classical straight-line Landau-Spitzer method
+        (`"classical"` or `"ls"`). The other 6 supported methods are `"lsmininterp"`,
+        `"lsfullinterp"`, `"lsclampmininterp"`, `"hlsmininterp"`, `"hlsmaxinterp"`, and 
+        `"hlsfullinterp"`. Please refer to the Notes section of this 
+        docstring for more information, including about abbreviated aliases of these names.
 
     Returns
     -------
-    lnLambda : float or numpy.ndarray
-        An estimate of the Coulomb logarithm that is accurate to
-        roughly its reciprocal.
+    ln_Lambda : float or numpy.ndarray
+        The dimensionless Coulomb logarithm.
 
     Raises
     ------
-    ValueError
+    `ValueError`
         If the mass or charge of either particle cannot be found, or
         any of the inputs contain incorrect values.
 
-    UnitConversionError
+    `~astropy.units.UnitConversionError`
         If the units on any of the inputs are incorrect.
 
         If the n_e, T, or V are not Quantities.
 
-    PhysicsError
+    `~plasmapy.utils.exceptions.PhysicsError`
         If the result is smaller than 1.
 
-    RelativityError
+    `~plasmapy.utils.exceptions.RelativityError`
         If the input velocity is same or greater than the speed
         of light.
 
     Warns
     -----
-    ~astropy.units.UnitsWarning
-        If units are not provided, SI units are assumed
+    : `~astropy.units.UnitsWarning`
+        If units are not provided, SI units are assumed.
 
-    ~plasmapy.utils.RelativityWarning
+    : `~plasmapy.utils.exceptions.RelativityWarning`
```

Comment:
Yes, I noticed that colons are used in this way in the docstring of `Alfven_speed` when I compared it with the docstring of `Coulomb_logarithm` for reference. Without colons, the warnings are displayed in the docs as literal text, "~astropy.units.UnitsWarning" and "~plasmapy.utils.RelativityWarning", rather than as code that is hyperlinked to the documentation of the warnings. I didn't find any information online on why the colons work. Maybe @StanczakDominik or @rocco8773 knows?

Reply:
Interesting, thanks!